### PR TITLE
Add rke2 standalone install script for Windows

### PR DESCRIPTION
**Pull request by mdrahman-suse**
#### Proposed Changes ####

This PR includes a script that automates installing RKE2 in Windows instances to reduce the manual effort needed for setup and installation. 

That can be decided based on feedback from the team

#### Types of Changes ####

No changes to the RKE2 codebase 

#### Verification ####

For testing,
- Setup a server node and install rke2
- Setup an Windows instance
- Download the script
- Run the script by providing serverIP, token and rke2 version

#### Linked Issues ####

https://github.com/rancher/qa-tasks/issues/151

### CODE REVIEW

**Comment by brandond** [comment]
Can we maybe call this a "quickstart" script (and name it similarly) to avoid any confusion around the intent and purpose of this script? We already have an install script for Windows, but the RKE2 installers for both Linux and Windows intentionally don’t handle configuration or start the service. That will probably make documenting it easier as well.

File: windows/rke2-standalone-install.ps1 [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    elseif ($Version) {
+        $Version = $Version
+    }
+    return $Version
+}
+
+function Enable-Features() {
+    Enable-WindowsOptionalFeature -Online -FeatureName Containers -All
+}
```
Comment (dereknola) [inline windows/rke2-standalone-install.ps1:63]: Doesn't the node need to restart after this is enabled?

Comment (brandond) [reply windows/rke2-standalone-install.ps1]: I think there is a -Restart flag that will reboot it if necessary. If this is our YOLO script we might as well add that?

Comment (mdrahman-suse) [reply windows/rke2-standalone-install.ps1]: As per the doc, it is required to reboot. But we saw rke2 was installed and started successfully even with no restart. So need to check if that is a hard requirement. I will check on this, thanks!

File: windows/rke2-quickstart.ps1 [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+    elseif ($Version) {
+        $Version = $Version
+    }
+    return $Version
+}
+
+function Enable-Features() {
+    Enable-WindowsOptionalFeature -Online -FeatureName Containers -All
+    Restart-Computer -Wait -For PowerShell -Timeout 300 -Delay 2
```
Comment (brandond) [inline windows/rke2-quickstart.ps1:64]: It looks like prompting to reboot is actually the default, so unconditionally rebooting probably isn't the right answer.

https://learn.microsoft.com/en-us/powershell/module/dism/enable-windowsoptionalfeature?view=windowsserver2022-ps

Comment (mdrahman-suse) [reply windows/rke2-quickstart.ps1:64]: So when rke2 joins successfully, the system reboots automatically.. removing this Restart command should be fine then?

Comment (mdrahman-suse) [reply windows/rke2-quickstart.ps1:64]: I have tested with both with and without this command (single node and HA) and was successfully able to join the windows node to the cluster

**Review by rancher-max — APPROVED** [review]

File: windows/rke2-quickstart.ps1 (hunk 2 above)
Comment (mdrahman-suse) [reply windows/rke2-quickstart.ps1]: Removed this Restart command. Will address any issues if observed on a separate PR

**Review by brandond — APPROVED** [review]

**Comment by mdrahman-suse** [comment]
Merging in 5 mins if no concerns

**Pull request merged**